screens/Login.tsx

```diff
@@ -44,7 +45,11 @@ const Login = ({ navigation }: Props) => {
         }
       })
 
-      console.log(response.data.data);
+      // console.log(response.data.data);
+      await Storage.storeData('token', response.data.data['access_token']);
+      await Storage.storeData('renewal_token', response.data.data['renewal_token']);
+      await Storage.storeData('token_expiry_time', response.data.data['token_expiry_time']);
+
```

Comment:
Let's store the whole object in a single JSON string

```suggestion
      await Storage.storeData('session', JSON.stringify(response.data.data);
```